sandbox/pages/SandboxPop.vue

```diff
@@ -133,6 +133,73 @@
           </KPop>
         </KComponent>
       </SandboxSectionComponent>
+      <SandboxSectionComponent title="offset">
+        <div class="horizontal-container">
+          <KPop
+            button-text="auto"
+            :offset="KUI_SPACE_100"
+            placement="auto"
+          >
+            <template #content>
+              Popover content.
+            </template>
+          </KPop>
+          <KPop
+            button-text="top"
+            :offset="KUI_SPACE_100"
+            placement="top"
+          >
+            <template #content>
+              Popover content.
+            </template>
+          </KPop>
+          <KPop
+            button-text="bottom"
+            :offset="KUI_SPACE_100"
+            placement="bottom"
+          >
+            <template #content>
+              Popover content.
+            </template>
+          </KPop>
+          <KPop
+            button-text="left"
+            :offset="KUI_SPACE_100"
+            placement="left"
+          >
+            <template #content>
+              Popover content.
+            </template>
+          </KPop>
+          <KPop
+            button-text="right"
+            :offset="KUI_SPACE_100"
+            placement="right"
+          >
+            <template #content>
+              Popover content.
+            </template>
+          </KPop>
+          <KPop
+            button-text="top-start"
+            :offset="KUI_SPACE_100"
+            placement="top-start"
+          >
+            <template #content>
+              Popover content.
+            </template>
+          </KPop>
+          <KPop
+            button-text="bottom-end"
+            :offset="KUI_SPACE_100"
+            placement="bottom-end"
+          >
+            <template #content>
+              Popover content.
+            </template>
+          </KPop>
```

Comment:
nit/suggestion: could you update the button text to display the offset, e.g. "Offset 100px"

And then also maybe add a few examples of different offsets?